### ISSUE TITLE
Add default backend for Polaris ingress

### ISSUE THREAD

**Issue by CharmanderJieniJieni** [reactiveops/charts#1129]
**Is your feature request related to a problem? Please describe.**
GKE requires default backend to be configured for container native load balancer, otherwise it returns 404. https://cloud.google.com/kubernetes-engine/docs/concepts/ingress#default_backend

**Describe the solution you'd like**
Add default backend configurable in https://github.com/FairwindsOps/charts/blob/master/stable/polaris/templates/ingress.yaml based in variable, or just default to https://github.com/FairwindsOps/charts/blob/b1a47fc8c3b6accbf6826fa126223196623fe179/stable/polaris/templates/ingress.yaml#L35

**Additional context**
I tried add below section to rendered helm chart to get it work on container native load balancer in GKE

```yaml
spec:
  defaultBackend:
    service:
      name: polaris-dashboard
      port:
        number: 80
  rules:
  - http:
      paths:
      - backend:
          service:
            name: polaris-dashboard
            port:
              number: 80
        path: /
        pathType: Prefix
```

**Comment by sudermanjr** [comment]
Interesting, I wasn't familiar with this configuration. Seems like it would be straightforward enough to add an empty map value `ingress.defaultBackend: {}` to do this. Would you be willing to make a PR?

**Comment by CharmanderJieniJieni** [comment]
sorry for delay, get chance to work on this now, PR on the way

**Comment by CharmanderJieniJieni** [comment]
created https://github.com/FairwindsOps/charts/pull/1229

**Comment by transient1** [comment]
Fixed in #1229 above. Closing.
